worlds/hexcellsinfinite/docs/en_Hexcells Infinite.md

```diff
@@ -0,0 +1,18 @@
+# Hexcells Infinite
+
+## Where is the options page?
+
+The [player options page for this game](../player-options) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+Every level is randomized, akin to the built-in seeded level generator.
+
+## Which items can be in another player's world?
+
+Gems, which are the little blue pips you gain for beating a level in standard play. These are required to unlock levels.
+
+## When the player receives an item, what happens?
+
+You will need to use the AP Text Client to see items sent out and received.
```

Comment:
I think this is a bit of a weird way to write this. It might make more sense to say something like "X thing will happen. No message will be shown in game, so it's recommended to use the Archipelago Text Client to see when are items sent or received."